ontology/yaml/resources/LIGHTING/entity_types/GENERALTYPES.yaml

```diff
@@ -14,43 +14,39 @@
 
 LT:
   description: "Luminaire, lighting fixture."
+  is_abstract: true
   implements:
   - EQUIPMENT
-  uses:
-  - dimmer_percentage_command # in % indicating brightness
-
-LPS:
-  description: "Lighting photocell sensor."
-  implements:
-  - MTS
-
-LMS:
-  description: "Lighting motion sensor."
-  implements:
-  - MTS
 
 LTGW:
   description: "Lighting gateway."
-  implements:
-  - EQUIPMENT
-  
-LAC:
-  description: "lighting area controller."
+  is_abstract: true
   implements:
   - EQUIPMENT
 
 LCM:
   description: "Lighting control module."
+  is_abstract: true
   implements:
   - EQUIPMENT
 
 LKP:
   description: "Lighting keypad."
+  is_abstract: true
   implements:
+  - EQUIPMENT
   - USER_INTERFACE
 
-LIGHTING_GROUP:
+LTB: # May need to add to BOS naming standard
+  description: "Battery that serves lighting devices. "
+  is_abstract: true
+  implements:
+  - EQUIPMENT
+
+LGRP:
+  is_abstract: true
   description: "A group containing one or more luminaires."
 
-LIGHTING_SCENE:
+LSCN:
```

Comment:
why do we need a type for this?

Reply:
@pisuke no idea. the concept of the scene seems undesirable to model to me

Reply:
I agree there is no need to model the scene as an entity

Reply:
conclusion: remove